release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -109,6 +109,13 @@ For more information, see xref:../networking/ingress-controller-dnsmgt.adoc#ingr
 
 For more information, see the xref:../networking/hardware_networks/about-sriov.adoc#supported-devices_about-sriov[supported devices].
 
+[id="ocp-4-12-switch-aws-load-balancer"]
+==== Switch between AWS load balancer types without deleting the Ingress Controller
+
+You can update the Ingress Controller to switch between an AWS Classic Load Balancer (CLB) and an AWS Network Load Balancer (NLB) without deleting the Ingress Controller. Previously, the Ingress Controller had to be force replaced which resulted in an expected outage that could last for several minutes.
```

Comment:
```suggestion
You can update the Ingress Controller to switch between an AWS Classic Load Balancer (CLB) and an AWS Network Load Balancer (NLB) without deleting the Ingress Controller. Previously, the Ingress Controller had to be force replaced, resulting in an expected outage that could last for several minutes.
```

**nitpick:** Looks okay without this change as well.

Reply:
thanks Gaurav! I ended up chopping this sentence 😄